structuraid_core.gemspec

```diff
@@ -42,6 +42,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubocop-rspec', '~> 2.16.0'
   spec.add_development_dependency 'simplecov', '~> 0.22.0'
 
+  spec.add_dependency 'matrix'
```

Comment:
Add a version constraint here please, to have more control on the version that would be installed

Reply:
ok let's doit by that way